src/cowboy_telemetry_h.erl

```diff
@@ -49,7 +49,17 @@ info(StreamID, Info, [Next0 | StartTime]) ->
     end,
     {Commands, [Next | StartTime]}.
 
-terminate(StreamID, Reason, [Next | _]) ->
+terminate(StreamID, Reason, [Next | StartTime]) ->
+    case Reason of
+        {socket_error, _, _} = Reason ->
```

Comment:
Could match even more specifically on `{socket_error, closed, _}` since `socket_error` is also used in other situations

https://github.com/ninenines/cowboy/blob/39b2816255503910dc23e2fdf703ee63bbc8953e/src/cowboy_http.erl#L230-L233

Reply:
I think it makes sense, but does it mean we emit `exception` in other cases?

Reply:
I think there is a chance that you send a response and then the socket is closed. So just to be safe, I would change the `info` callback to return `[Next | done]` as the State and if the StartTime is `done ` here, you know you shouldn't log it. WDYT?

Reply:
sounds great!